test_runner/src/test/kotlin/ftl/run/GenericTestRunnerTest.kt

```diff
@@ -31,7 +33,7 @@ class GenericTestRunnerTest {
         val result = createMock(1).beforeRunMessage(
             listOf(Chunk(listOf(TestMethod(name = "", time = 0.0))))
         ).normalizeLineEnding()
-        assert(result, "  1 test / 1 shard\n")
+        assert(result, "  1 test / 1 shard".withNewLineAtTheEnd().startWithNewLine())
```

Comment:
can not the same be achieved with
```suggestion
        assert(result.trim(), "  1 test / 1 shard".trim())
```

Reply:
replaced